From 5b64c3085942856f520823c12c871a4d96b75e35 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Wed, 6 Dec 2017 15:48:50 +0000
Subject: [PATCH] v2v: docs: Remove min version of VDDK, recommend nbdkit >=
 1.1.25 (RHBZ#1513884).

Requires -> recommends because older versions of nbdkit will likely
work also.

See also:
https://bugzilla.redhat.com/show_bug.cgi?id=1513884#c3

(cherry picked from commit 5ac24b490ebcf2e16e4ee2284a7f59bcc47cc808)
---
 v2v/virt-v2v.pod | 14 +++++---------
 1 file changed, 5 insertions(+), 9 deletions(-)

diff --git a/v2v/virt-v2v.pod b/v2v/virt-v2v.pod
index 417c25331..13f021c6d 100644
--- a/v2v/virt-v2v.pod
+++ b/v2v/virt-v2v.pod
@@ -1304,13 +1304,9 @@ library is permitted by the license.
 
 =item 2.
 
-VDDK E<ge> 6.5 is required.
-
-=item 3.
-
-You must also compile nbdkit, enabling the VDDK plugin.  At least
-nbdkit E<ge> 1.1.16 is required, but it is usually best to compile
-from the git tree.
+You must also compile nbdkit, enabling the VDDK plugin.  nbdkit E<ge>
+1.1.25 is recommended, but it is usually best to compile from the git
+tree.
 
 =over 4
 
@@ -1335,13 +1331,13 @@ eg:
 
  export PATH=/path/to/nbdkit-1.1.x:$PATH
 
-=item 4.
+=item 3.
 
 You must find the SSL "thumbprint" of your VMware server.  How to do
 this is explained in L<nbdkit-vddk-plugin(1)>, also available at the
 link above.
 
-=item 5.
+=item 4.
 
 VDDK imports require a feature added in libvirt E<ge> 3.7.
 
-- 
2.14.3

